flaskserver.py

```diff
@@ -6,14 +6,17 @@
 from .pindb import PINDb
 from wallycore import hex_from_bytes, hex_to_bytes, AES_KEY_LEN_256, \
     AES_BLOCK_LEN
+from dotenv import load_dotenv
 
 b2h = hex_from_bytes
 h2b = hex_to_bytes
 
 # Time we will retain active sessions, in seconds.
 # ie. maximum time allowed 'start_handshake' (which creates the session)
 # and the get-/set-pin call, which utilises it.
-SESSION_LIFETIME = 30
+# Can be set in environment, defaults to 30s
+load_dotenv()
+SESSION_LIFETIME = os.environ.get(int(os.environ.get('SESSION_LIFETIME', 30)))
```

Comment:
I think you have an extra call to `os.environ.get` here? Only the inner `int()` expression seems needed...

Reply:
yy daft-o